homeassistant/components/binary_sensor/xiaomi_aqara.py

```diff
@@ -311,6 +316,48 @@ def parse_data(self, data, raw_data):
             return False
 
 
+class XiaomiVibration(XiaomiBinarySensor):
+    """Representation of a Xiaomi Vibration Sensor."""
+
+    def __init__(self, device, name, data_key, hass, xiaomi_hub):
```

Comment:
Don't pass in `hass`. It will be set on the entity when it has been added to home assistant.

Reply:
@MartinHjelmare I copied another sensor (XiaomiButton from the same file) to create this one. And, I'll admit, Python isn't my strongest language. I understand entirely what you mean by "don't pass in hass" and I can easily remove it from the __init__ signature. But, two lines later there is a "self._hass = hass" which will surely fail, but can probably also just be removed. And then later in the class, self._hass is used as "self._hass.bus.fire()". I'm not sure what to change that last call to. 

Also, do I need to remove "hass" from the constructor of all the other classes in this file, or just the new one that this PR is attempting to add?